packages/noir-ethereum-api/src/noir/encode.ts

```diff
@@ -35,14 +35,14 @@ export function encodeBytes(value: bigint, length: number) {
   return encodeHex(`0x${hexValue}`);
 }
 
-export function encodeHex(hexString: string) {
+export function encodeHex(hexString: string, compactHexValue: boolean = true) {
```

Comment:
Why do we need it?

Reply:
We have a lot of test data that is in this format